lib/adt.ex

```diff
@@ -56,12 +56,8 @@ defmodule ADT do
   end
 
   # Maps a module like Foo.Bar.Baz into a short string "Baz"
-  def _shorten_module_name(variant) do
-    Regex.named_captures(~r/\.(?<short>[^\.]+)$/, inspect(variant), include_captures: true) |> Map.fetch!("short")
-  end
-
-  def _shorten_match(match) do
-    Regex.named_captures(~r/\.(?<short>[^\.]+){/, inspect(match), include_captures: true) |> Map.fetch!("short")
+  def _shorten(name) do
+    Regex.named_captures(~r/\.(?<short>[^\.{]+)($|{)/, inspect(name), include_captures: true) |> Map.fetch!("short")
```

Comment:
The dot needs not be escaped in the character class(...or elixir's regexes are weird..)